docs/app/documentation/component-docs/tabs/tabs-docs.component.html

```diff
@@ -25,6 +25,17 @@ <h2>Complex Tab Titles</h2>
 </component-example>
 <code-example [code]="complexH" [language]="'HTML'"></code-example>
 
+<separator></separator>
+<h2>Tab Navigation Mode</h2>
+<description>
+    There is possibility to change tab mode to navigation, it means that there is not possibility to put content, but there is possibility to add some
```

Comment:
It is possible to use the tabs in navigation mode. While there is no option to provide some content in this mode, it does allow the use of a router-outlet with <code>[routerLink]</code> or <code>[href]</code> attributes.

Reply:
Also I think we could use a more detailed description of a practical use case here

Reply:
Docs changed.